sky/serve/load_balancer.py

```diff
@@ -223,9 +225,15 @@ async def startup():
         uvicorn.run(self._app, host='0.0.0.0', port=self._load_balancer_port)
 
 
-def run_load_balancer(controller_addr: str, load_balancer_port: int):
+def run_load_balancer(controller_addr: str, load_balancer_port: int,
+                      policy_name: str):
+    if policy_name == 'round_robin':
+        policy = lb_policies.RoundRobinPolicy()
+        raise ValueError(f'Unknown load balancing policy: {policy_name}')
```

Comment:
For this, we might want to refer to the recovery strategy implementation:

1. Specify a registry from name to the class (RECOVERY_STRATEGIES in this example);
2. Define an function in the base class to choose from this dictionary.

https://github.com/skypilot-org/skypilot/blob/292febc85cee658bd4370dd1894f2a6c97a5e264/sky/jobs/recovery_strategy.py#L85-L112

This would help up to keep track of all lb policy in one places ;)